src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -3,9 +3,18 @@
 public class DefaultCountingOutRhymer
 {
 	
-	private int[] numbers = new int[12];
+	private static final int FULL_CONSTANT = 11;

```

Comment:
This constant is related to MAX_SIZE. Changing only one leads to an error (inconsistency).